salt/_multibuild

```diff
@@ -0,0 +1,3 @@
+<multibuild>
+  <flavor>test</flavor>
```

Comment:
Could we make the flavor name more descriptive? E.g.:

```suggestion
  <flavor>testsuite_included</flavor>
```

(Not sure if there's an RPM convention though, to include flavor names such as `test`/`debug`/..., in which case feel free to disregard)

Reply:
`test` is often used to enable running tests in `%check` , e.g. in https://build.opensuse.org/package/show/openSUSE:Factory/rust1.74.